taskipy/task_runner.py

```diff
@@ -20,40 +23,79 @@ def list(self):
         """lists tasks to stdout"""
         longest = len(max(self.project.tasks, key=len))
         for k, v in self.project.tasks.items():
-            print(f'{k:<{longest}}  {v}')
+            print(f'{k:<{longest}}  {v.description or v.command}')
```

Comment:
This could be a separate pull request, but I feel like it'd be more ideal to decouple this method from doing the printing and gathering of the tasks. We could just as easily return the string to print in `list()` and do a `print(runner.list())` in the `cli.py` file.

Reply:
following your comment I created a formatter that takes care of formatting the list. It does the following:
1. Indents the text
2. Handles wraps and newlines
3. Colors the command to create a clear distinction between commands and descriptions